packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -313,10 +309,10 @@ class FuchsiaDevice extends Device {
     }
 
     if (!prebuiltApplication) {
-      await buildFuchsia(fuchsiaProject: FlutterProject.current().fuchsia,
-                         targetPlatform: await targetPlatform,
-                         target: mainPath,
-                         buildInfo: debuggingOptions.buildInfo);
+      globals.printError(
+        'prebuiltApplication == false and this tool does not currently build apps',
```

Comment:
Will `prebuiltApplication == false` be clear to the flutter tool user (assuming they're the ones who see this message)? I'm not sure where `prebuiltApplication` come from.

Is there a flag they pass to configure `prebuiltApplication`? If so, should we use that flag's name here instead?

Reply:
Good idea, you can grab the flag name from: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/runner/flutter_command.dart#L121

Reply:
actually, I'm thinking this should probably be a toolexit, (e.g. `throwToolExit('Running Fuchsia apps requires the "${FlutterOptions.kUseApplicationBinary}" flag to be set to...');`), which means exit the tool with a non-zero code and error message, but the mistake was on the user's side, and thus don't send a report to crash reporting.

Reply:
done